gravitee-apim-portal-webui-next/src/scss/theme/variables.scss

```
@@ -14,37 +14,54 @@
  * limitations under the License.
  */
 
-/**
-Global variables
- */
+// ============================================================================
+// Global variables
+// ============================================================================
 
-$font-family: var(--gio-app-font-family);
+// -- Layout & Background --
 $background-color: var(--gio-app-background-color);
 $nav-bar-background-color: var(--gio-app-nav-bar-container-color, color-mix(in srgb, $background-color, transparent 20%));
 $card-background-color: var(--gio-app-card-background-color);
-$card-container-shape: var(--gio-app-card-container-shape, 4px);
-$button-container-shape: var(--gio-app-button-container-shape, 4px);
+
+// -- Typography --
+$font-family: var(--gio-app-font-family);
 $default-text-color: var(--gio-app-font-color, #1d192b);
 $default-text-contrast-color: var(--gio-app-font-contrast-color, #fff);
 $paragraph-color: color-mix(in srgb, $default-text-color 75%, transparent);
 
-// HSL values calculated in theme.service.ts
+// -- Shapes --
+$card-container-shape: var(--gio-app-card-container-shape, 4px);
+$button-container-shape: var(--gio-app-button-container-shape, 4px);
+
+// -- Theme Color Palette calculated by a theme service --
 $primary-main-color: var(--gio-app-primary-main-color);
 $secondary-main-color: var(--gio-app-secondary-main-color);
 $tertiary-main-color: var(--gio-app-tertiary-main-color);
 $error-main-color: var(--gio-app-error-main-color);
 
-/**
-Component variables
- */
+// ============================================================================
+// Component-Specific variables
+// ============================================================================
+
+// -- Banner --
 $banner-background-color: var(--mdc-list-list-item-leading-avatar-color);
-$banner-error-background-color: color-mix(in srgb, $error-main-color 16%, $card-background-color);
 $banner-text-color: var(--mdc-outlined-text-field-focus-label-text-color);
+$banner-error-background-color: color-mix(in srgb, $error-main-color 16%, $card-background-color);
 $banner-error-text-color: $error-main-color;
-$chip-background: var(--gio-app-chip-container-color, $card-background-color);
+
+// -- Table --
 $table-border-color: color-mix(in srgb, $secondary-main-color 35%, transparent);
 $table-header-background-color: var(--gio-app-table-header-container-color, $secondary-main-color);
 $table-border-radius: var(--gio-app-table-container-shape, 4px);
+
+// -- User Avatar --
+$user-avatar-height: var(--gio-app-user-avatar-height, 48px);
+$user-avatar-container-shape: var(--gio-app-user-avatar-container-shape, 50%);
+$user-avatar-border-color: var(--gio-app-user-avatar-border-color);
```

```suggestion
$user-avatar-border-color: var(--gio-app-user-avatar-outline-color);
```
I think this might match more of the Material vocabulary 🤔